File: Appfrontend/src/app/models/peliculaDTO.ts
export interface PeliculaDTO {
  id_Pelicula: number;
  nombre: string;
  duracion: number;
  imagen?: string; 
  activo: boolean;
}

export interface PeliculaCreateDTO {
  nombre: string;
  duracion: number;
  imagen?: string;
}

export interface PeliculaUpdateDTO {
  id_Pelicula: number;
  nombre: string;
  duracion: number;
  imagen?: string;
}

File: API-Cine/API-Cine/Models/DTOs/PeliculaDTO.cs
namespace API_Cine.Models.DTOs
{
    public class PeliculaDTO
    {
        public int Id_Pelicula { get; set; }
        public string Nombre { get; set; }
        public int Duracion { get; set; }
        public string? Imagen { get; set; }
        public bool Activo { get; set; }
    }

    public class PeliculaCreateDTO
    {
        public string Nombre { get; set; }
        public int Duracion { get; set; }
        public string? Imagen { get; set; }
    }
    public class PeliculaUpdateDTO
    {
        public int Id_Pelicula { get; set; }
        public string Nombre { get; set; }
        public int Duracion { get; set; }
        public string? Imagen { get; set; }

    }
}
